components/XOGame.js

```diff
@@ -0,0 +1,140 @@
+import styled from "styled-components";
+import {useState} from "react";
+
+const players = {
+
+  HUMAN: {
+  SYM: "X",
+  NAME: "You",
+  },
+};
+
+export default function XOGame() {
+  
+  const [board, setBoard] = useState([
+    ["", "", ""],
+    ["", "", ""],
+    ["", "", ""],
+  ]);
+
+  function playFieldNr(arrayIndex, index) {
```

Comment:
This is better than before, but another developer (especially from another country) would still not know what this function actually does